### PR TITLE
DisplayInformation properties on Android and iOS

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #385 

## PR Type
What kind of change does this PR introduce?
- Feature

## What is the current behavior?
Support of `DisplayInformation` properties is limited.

## What is the new behavior?
Added support for the following `DisplayInformation` properties on iOS and Android:

- `NativeOrientation`
- `LogicalDpi`
- `RawDpiX`
- `RawDpiY`
- `ResolutionScale`
- `StereoEnabled`
- `RawPixelsPerViewPixel`
- `DiagonalSizeInInches`
- `ScreenHeightInRawPixels`
- `ScreenWidthInRawPixels`

I will work on adding the `DisplayInformation` events in a separate PR.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

Internal Issue (If applicable): #385

### CODE REVIEW

**Comment by MartinZikmund** [comment]
Found out the `CurrentOrientation` property is not working properly on Android. I will fix it now.

**Comment by MartinZikmund** [comment]
Good, `CurrentOrientation` and `OrientationChanged` are now implemented on Android.

**Comment by MartinZikmund** [comment]
@MatFillion _do-not-merge_ can be removed now

File: src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs [hunk 1]
```diff
@@ -1,22 +1,184 @@
...
+			var portraitOrientation0Or180 =
+				(rotation == SurfaceOrientation.Rotation0 ||
+				rotation == SurfaceOrientation.Rotation180)
+				&& height > width;
+			var portraitOrientation90Or270 =
+				(rotation == SurfaceOrientation.Rotation90 ||
+				rotation == SurfaceOrientation.Rotation270)
+				&& width > height;
+
+			// if the device's natural orientation is portrait:
```
Comment (davidjohnoliver) [inline src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs]: Could you just use the NativeOrientation property here? Why is the logic slightly different?

File: src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs [hunk 2]
```diff
@@ -1,22 +1,184 @@
...
+						break;
+				}
+			}
+		}
+
+
+		internal void HandleConfigurationChange()
+		{
+			var previousOrientation = CurrentOrientation;
+			Initialize();
```
Comment (davidjohnoliver) [inline src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs]: Presumably only CurrentOrientation needs to be updated every time?

Comment (MartinZikmund) [reply src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs]: No, `HandleConfigurationChange` can be triggered by `SizeChange` on Android, which can be something like switching to a completely new display, etc. Hence any of the properties could potentially change

File: src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs]: You are right, this could probably be simplified/unified, I will take a look

Comment (MartinZikmund) [reply src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs]: Done :-)

**Comment by MartinZikmund** [comment]
@jeromelaban What does _/azp run_ mean :-) ?

**Comment by jeromelaban** [comment]
@MartinZikmund it's to re-run a build when a PR is from an external contributor: https://go.microsoft.com/fwlink/?linkid=2056279

**Comment by MartinZikmund** [comment]
I just pushed the fix for the review above, so now the changes should be complete

**Pull request merged**